{
  "comments": [
    {
      "key": {
        "uuid": "dabed58f_9579c795",
        "filename": "ci/envs/host-config",
        "patchSetId": 10
      },
      "lineNbr": 37,
      "author": {
        "id": 473
      },
      "writtenOn": "2017-02-16T01:27:47Z",
      "side": 1,
      "message": "will this function be called only when something wrong? I noticed it\u0027s called when success, so it basically means exit, right?",
      "revId": "be208cb48be4f4db438ce27be15e1f6166c26910",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    }
  ]
}